daemon/api_aspects.go

```diff
@@ -90,13 +100,28 @@ func setAspect(c *Command, r *http.Request, _ *auth.UserState) Response {
 	st.Lock()
 	defer st.Unlock()
 
+	schema := aspects.NewJSONSchema()
+	getter := bagGetter(st, account, bundleName)
+	setter := func(bag aspects.JSONDataBag) error {
+		return updateDatabags(st, account, bundleName, bag)
+	}
+
+	tx, err := aspects.NewTransaction(getter, setter, schema)
+	if err != nil {
+		return toAPIError(err)
+	}
+
```

Comment:
this kind of code belongs to a helper in aspectstate for consistency of responsability